### PR TITLE
chore(config): add config for zappr, editorconfig, eslint, travis

### LINKED ISSUE #8: Create open source repo for tracking
- Create this repo
- Assign permissions
- Configure Zappr

### PR DESCRIPTION
**Pull request by cjhowald**
Update .gitignore and add config for zappr & others.

Fixes #8

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+language: node_js
+node_js:
+  - 'stable'
+
+install:
+  - npm install -g gulp bower
+  - bower install
```
Comment (gregdeane) [inline .travis.yml]: Let's take out the `bower`/`gulp` references.

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  - npm install
+
+before_script:
+  - export DISPLAY=:99.0
+  - sh -e /etc/init.d/xvfb start
+
+script:
+  - npm run test
```
Comment (gregdeane) [inline .travis.yml]: Not a big deal but we could just say `npm test`

File: .travis.yml (hunk 1 above)
Comment (cjhowald) [inline .travis.yml]: Good call. Fixed.

**Comment by gregdeane** [comment]
👍

**Comment by cjhowald** [comment]
👍

**Pull request merged**